assets/js/blocks/cart-checkout/checkout/inner-blocks/checkout-terms-block/editor.scss

```
@@ -14,8 +14,13 @@
 
 .wc-block-components-checkbox {
 	margin-top: 0;
+
+	&.wc-block-checkout__create-account {
+		margin-top: 1em;
+	}
 }
```

> I wonder if, long term, the best solution is adding exceptions to the margin rule instead of maybe taking a more generic approach. For now, I think this is totally fine! 👍

I had the same thoughts, @sunyatasattva. What do you think about changing it to:

```suggestion
.wp-block-woocommerce-checkout-terms-block .wc-block-components-checkbox {
	margin-top: 0;
}
```

That way, the CSS definition only targets the element that you intend to change in #5191.

---

I'm actually not sure that the best way is to target the checkbox. The problem is that the checkbox is often associated with a label. If the label wraps to 2 lines, the alignment gets out of whack.

I think we are thinking it backwards. It's probably not the checkbox which needs any margin, but the input that needs a bottom margin. Or perhaps it's the ToS block which uses a strange and unorthodox HTML structure (the label is a `div` next to the checkbox container).

In any case, my point here is that if we keep targeting the margin of the checkbox we might go on playing a whack-a-mole situation.

---

> What about adding bottom margin to the text input containers? Can we try that and see if that doesn't create any disaster? Because of how [margin collapsing](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing) works, that shouldn't be a disaster.

I looked up this option and currently, all input fields, except the input fields of the first row of each section, use a `margin-top`. Adding a general `margin-bottom` affects all input fields. 

I see two potential solutions:

1. Add `margin-bottom` to the email address input field.
2. Add `margin-top` to the create an account div.

Funny enough, the second option, is my initial fix. Do you want to take another look, @sunyatasattva, or shall we go ahead and merge my initial solution?

---

I don't want to stall you forever on this. I had seen about the `margin-top`: splitting it by two between top and bottom did the trick in my testing.

But, again, I don't know if this is worth stalling this issue even more: if we find any problem later, we'll refer to it and discuss a more robust solution, maybe. For now, I'm going to approve it :)